Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/converter/XMLGregorianConverter.java

```diff
@@ -58,35 +58,38 @@ public Object getAsObject(FacesContext context, UIComponent component, String ne
 
     @Override
     public String getAsString(FacesContext context, UIComponent component, Object value) {
-        Map<String, Object> attributes = component.getAttributes();
-        if (attributes.containsKey("pattern")) {
-            pattern = (String) attributes.get("pattern");
-        }
-        setPattern(pattern);
-        if (attributes.containsKey("locale")) {
-            locale = (Locale) attributes.get("locale");
-        }
-        setLocale(locale);
-        if (attributes.containsKey("timeZone")) {
-            timeZone = (TimeZone) attributes.get("timeZone");
-        }
-        setTimeZone(timeZone);
-        if (attributes.containsKey("dateStyle")) {
-            dateStyle = (String) attributes.get("dateStyle");
-        }
-        setDateStyle(dateStyle);
-        if (attributes.containsKey("timeStyle")) {
-            timeStyle = (String) attributes.get("timeStyle");
-        }
-        setTimeStyle(timeStyle);
-        if (attributes.containsKey("type")) {
-            type = (String) attributes.get("type");
-        }
-        setType(type);
+        if (null != value && null != component && null != component.getAttributes()) {
```

Comment:
break this into 2-if-statement if you agreen

if (null != component && null != component.getAttributes()) {